ansible/roles/jupyterhub/templates/jupyterhub_config_base.py.j2

```diff
@@ -101,8 +103,9 @@ base_url = os.environ.get('JUPYTERHUB_BASE_URL') or ''
 c.JupyterHub.base_url = base_url
 
 # Shutdown servers on logout
+# convert string to boolean if the value comes from the env var
 shutdown_on_logout = os.environ.get('JUPYTERHUB_SHUTDOWN_ON_LOGOUT') or True
```

Comment:
To avoid a type error with the **strtobool** function the literal value `True` must be also enclose as string

Reply:
Good point, we can't assume that the env var will always have a value associated with it.